frontend/src/metabase/redux/downloads.ts

```diff
@@ -164,9 +164,9 @@ const getDatasetParams = ({
 }: DownloadQueryResultsOpts): DownloadQueryResultsParams => {
   const cardId = question.id();
 
-  const exportParams = {
-    format_rows: enableFormatting,
-    pivot_results: enablePivot,
+  const exportParams: Record<string, string> = {
+    format_rows: String(enableFormatting),
```

Comment:
Since we're spreading this into `URLSearchParams` we need to make the value a string otherwise, TypeScript would complain.